app/models/automated_checks/update_admin_claim_tasks_with_dqt.rb

```diff
@@ -0,0 +1,57 @@
+module AutomatedChecks
+  class UpdateAdminClaimTasksWithDqt
+    def initialize(
+      claim:,
+      dqt_teacher_status:,
+      admin_user: nil
```

Comment:
As we discussed, it would be better longer term to refactor users altogether so we can have DFE and system users but that's not in scope.

The current issue with this code isn't really related to visibility because we have precisely one `nil` user which we know is the system. It's actually do we need to mention user here at all?